```
======================================================================
FAIL: test_project_description_web_user (corehq.apps.hqwebapp.tests.test_views.TestBugReport)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/hqwebapp/tests/test_views.py", line 86, in test_project_description_web_user
    self.assertEqual(domain_object.project_description, 'Great NGO, Just Great')
AssertionError: None != 'Great NGO, Just Great'
-------------------- >> begin captured logging << --------------------
```

--- a/corehq/apps/hqwebapp/tests/test_views.py
+++ b/corehq/apps/hqwebapp/tests/test_views.py
@@ -20,26 +20,28 @@ class TestEmailAuthenticationFormAutocomplete(BaseAutocompleteTest):
 class TestBugReport(TestCase):
     domain = 'test-bug-report'
 
-    def setUp(self):
+    @classmethod
+    def setUpClass(cls):
         delete_all_users()
-        self.project = create_domain(self.domain)
-        self.web_user = WebUser.create(
-            self.domain,
+        cls.project = create_domain(cls.domain)
+        cls.web_user = WebUser.create(
+            cls.domain,
             'bug-dude',
             password='***',
         )
-        self.web_user.is_superuser = True
-        self.web_user.save()
-        self.commcare_user = CommCareUser.create(
-            self.domain,
+        cls.web_user.is_superuser = True
+        cls.web_user.save()
+        cls.commcare_user = CommCareUser.create(
+            cls.domain,
             'bug-kid',
             password='***',
         )
-        self.url = reverse("bug_report")
+        cls.url = reverse("bug_report")
 
-    def tearDown(self):
+    @classmethod
+    def tearDownClass(cls):
         delete_all_users()
-        self.project.delete()
+        cls.project.delete()
 
     def _default_payload(self, username):
         return {

--- a/corehq/apps/hqwebapp/views.py
+++ b/corehq/apps/hqwebapp/views.py
@@ -487,7 +487,7 @@ def bug_report(req):
     domain_object = Domain.get_by_name(report['domain'])
     current_project_description = domain_object.project_description
     new_project_description = req.POST.get('project_description')
-    if (req.couch_user.is_domain_admin() and
+    if (req.couch_user.is_domain_admin(domain=report['domain']) and
             new_project_description and
             current_project_description != new_project_description):
 

